Disable panic=abort to fix Yocto build error.
Disable strip=symbols because Yocto already strips the symbols and in the debug
environment we might want to use them.
diff --git a/Cargo.toml b/Cargo.toml
index 5a4be7400..8c59708a4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,5 +23,3 @@ repository = "https://github.com/thin-edge/thin-edge.io"
 codegen-units = 1
 lto = true
 opt-level = "z"
-panic = "abort"
-strip = "symbols"
